### PR TITLE
Final touches task1

### PR DESCRIPTION
**Pull request by fia0**
Some final changes including a few comments and some feedback to the course. If anyone wants to add to it, this PR will be open until this evening, you can go ahead and push on this PR with your own feedback.

### CODE REVIEW

File: task1/feedback.txt [hunk 1]
```diff
@@ -2,13 +2,19 @@ Please specify the amount of time you needed in hours.
...
-Understandable:  [ ] unclear   [ ] a bit unclear  [ ] clear
+Difficult:       [X] too easy  [ ] spot-on        [ ] too hard
+Instructive:     [X] a little  [ ] okay           [ ] a lot
+Understandable:  [ ] unclear   [ ] a bit unclear  [X] clear
 
 Comments:
 - Amount of time required too high? Did you have problems anywhere?
-- What was too hard or unclear?
 - What could we improve?
-For the love of god, when defining a 2 dimensional something use the x componend for left to right and y for up and down.
```
Comment (Cookie42) [inline task1/feedback.txt:13]: I want to rant man ..

File: task1/map.c [hunk 2]
```diff
@@ -15,22 +15,26 @@ enum cardd map[9];
 // The function set_dir should put the value dir at position x, y into the array map
 // Check x, y and dir for validity
 
-// i would like to note that i switched x and y here because i cant let x go downwards and y to the left, it just feels wrong
+// I would like to note that i switched x and y here because i cant let x go downwards and y to the left, it just feels wrong
```
Comment (SkyRiderSCII) [inline task1/map.c:18]: Does not the y component goes from left to right. Therefore y goes to the right.

File: task1/feedback.txt (hunk 1 above)
Comment (fia0) [reply task1/feedback.txt:13]: I left the rant included in the source file itself xD

**Review by SkyRiderSCII — APPROVED** [review]

File: task1/map.c (hunk 2 above)
Comment (fia0) [reply task1/map.c:18]: Technically yeah, but i'll let it stay like this for now as it's further explained a few more lines further down

**Review by darshit-97 — APPROVED** [review]

**Review by Cookie42 — APPROVED** [review]

**Pull request merged**